src/test/java/mops/presentation/SecurityContextUtil.java

```diff
@@ -0,0 +1,36 @@
+package mops.presentation;
```

Comment:
move this to another package

Reply:
I disagree, it is part of the all controller tests.